analyzers/tests/SonarAnalyzer.UnitTest/Rules/EmptyMethodTest.cs

```diff
@@ -78,45 +78,20 @@ public class EmptyMethodTest
         public void EmptyMethod_VB() =>
             Verifier.VerifyAnalyzer(@"TestCases\EmptyMethod.vb", new VB.EmptyMethod());
 
-        [DataTestMethod]
-        [DataRow(ProjectType.Product)]
-        [DataRow(ProjectType.Unknown)]
-        public void EmptyMethod_WithVirtualOverride_RaisesIssueForMainAndUnknownProject_CS(ProjectType projectType) =>
-            Verifier.VerifyAnalyzer(@"TestCases\EmptyMethod.OverrideVirtual.cs",
-                new CS.EmptyMethod(),
-                options: ParseOptionsHelper.FromCSharp8,
-                additionalReferences: null,
-                sonarProjectConfigPath: TestHelper.CreateSonarProjectConfig(
-                    nameof(EmptyMethod_WithVirtualOverride_RaisesIssueForMainAndUnknownProject_CS),
-                    projectType));
+        [TestMethod]
+        public void EmptyMethod_WithVirtualOverride_RaisesIssueForMainProject_CS() =>
+            Verifier.VerifyAnalyzer(@"TestCases\EmptyMethod.OverrideVirtual.cs", new CS.EmptyMethod());
```

Comment:
_Idea:_ We should use UT for the `Product` vs `Unknown` behavior, we don't need full end-to-end integration for each rule. Like this, it will run in a `Product` mode. There will never be `Unknown` for the rule.